en/docs/develop/advanced-development/applying-security-to-a-proxy-service.md

```diff
@@ -31,8 +31,8 @@ Follow the instructions below to create a **WS-Policy** resource in your integra
    You can also find this security policy file under the **Registry Explorer** section in the sidebar. Double-click on the file to open it.  
    [![Registry Explorer]({{base_path}}/assets/img/integrate/apply-security/119130870/registry-explorer.png)]({{base_path}}/assets/img/integrate/apply-security/119130870/registry-explorer.png)
 
-!!! Note
-    If you want to change any other properties, you can edit via the **Source View**.
+    !!! Note
+        Currently, the **Design View** of the policy editor is not available in WSO2 MI for VSCode. You can edit the properties via the **Source View**.  
```

Comment:
A part of the design view is added recently. So shall we keep the previous note?